releng/org.aposin.mergeprocessor.targetdefinition/org.aposin.mergeprocessor.targetdefinition.target

```diff
@@ -1,43 +1,43 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><?pde version="3.8"?><target name="MergeProcessor" sequenceNumber="146">

```

Comment:
Eclipse orders the targetdefinition file on it's own. Even sorting manually does not help as it shuffles the whole order again. Developers should not find changes in the workspace when applying the target definition file.